keps/sig-auth/3766-referential-authorization/README.md

```diff
@@ -732,12 +693,19 @@ a moot point.
 
 ### Do We Need Label Selectors?
 
+#### Rationale
+We are leaving this feature out because it dramatically increases the complexity
+of these APIs (both for users and implementations). These APIs do allow
+authorizing all same-namespace references, and support a list of names for
+cross-namespace references.
+
+#### Context
 Previous Discussions:
 * https://github.com/kubernetes/enhancements/pull/3767#discussion_r1084492070
 * https://github.com/kubernetes/enhancements/pull/3767#discussion_r1084674648
 
-As a natural next step, instead of simply allowing all, we could use
-label selectors to enable:
+As a natural next step, instead of simply allowing a named list of resources, we
+could use label selectors to enable:
 
 1. Access to resources with specific labels
```

Comment:
I guess this essentially and practically would force us to build the labelSelector feature into RBAC?

Reply:
+1, I don't think that's really viable, or if it is, it could/should be done separately.